mod/system_log/classes/Elgg/SystemLog/Logger.php

```diff
@@ -46,7 +46,7 @@ public static function listen(\Elgg\Event $event): void {
 		// check if this event is part of a sequence and can't be logged yet
 		// so we can delay the logging to the :after event
 		$sequence_id = $event->getSequenceID();
-		if (isset($sequence_id) && strpos($name, ':after') === false && strpos($name, ':before')) {
+		if (isset($sequence_id) && !str_contains($name, ':after') && strpos($name, ':before')) {
```

Comment:
can we make the strpos($name, ':before') more explicit? What are we checking here... i think str_contains

Reply:
changed